h2o-dist/index.html

```diff
@@ -761,4 +755,44 @@ <h2>Booklets</h2>
       </div>
     </div>
   </body>
+  <script type="text/javascript">
```

Comment:
By putting the script at the end of the page, we can avoid using jQuery to tell us when the DOM is ready.